api/bazel/repository_locations.bzl

```diff
@@ -1,66 +1,110 @@
+        project_desc = "Language Independent Interface Types For OpenCensus",
+        project_url = "https://github.com/census-instrumentation/opencensus-proto",
+        version = "0.3.0",
+        sha256 = "b7e13f0b4259e80c3070b583c2f39e53153085a6918718b1c710caf7037572b0",
+        strip_prefix = "opencensus-proto-{version}/src",
+        urls = ["https://github.com/census-instrumentation/opencensus-proto/archive/v{version}.tar.gz"],
+        last_updated = "2020-06-20",
+        use_category = ["api"],
     ),
     prometheus_metrics_model = dict(
-        sha256 = PROMETHEUS_SHA,
-        strip_prefix = "client_model-" + PROMETHEUS_GIT_SHA,
-        urls = ["https://github.com/prometheus/client_model/archive/" + PROMETHEUS_GIT_SHA + ".tar.gz"],
-    ),
-    opencensus_proto = dict(
-        sha256 = OPENCENSUS_PROTO_SHA256,
-        strip_prefix = "opencensus-proto-" + OPENCENSUS_PROTO_RELEASE + "/src",
-        urls = ["https://github.com/census-instrumentation/opencensus-proto/archive/v" + OPENCENSUS_PROTO_RELEASE + ".tar.gz"],
+        project_name = "Prometheus client model",
+        project_desc = "Data model artifacts for Prometheus",
+        project_url = "https://github.com/prometheus/client_model",
+        version = "60555c9708c786597e6b07bf846d0dc5c2a46f54",
+        sha256 = "6748b42f6879ad4d045c71019d2512c94be3dd86f60965e9e31e44a3f464323e",
+        strip_prefix = "client_model-{version}",
+        urls = ["https://github.com/prometheus/client_model/archive/{version}.tar.gz"],
+        last_updated = "2020-06-23",
+        use_category = ["api"],
     ),
     rules_proto = dict(
-        sha256 = RULES_PROTO_SHA256,
-        strip_prefix = "rules_proto-" + RULES_PROTO_GIT_SHA + "",
-        urls = ["https://github.com/bazelbuild/rules_proto/archive/" + RULES_PROTO_GIT_SHA + ".tar.gz"],
-    ),
-    com_github_openzipkin_zipkinapi = dict(
-        sha256 = ZIPKINAPI_SHA256,
-        strip_prefix = "zipkin-api-" + ZIPKINAPI_RELEASE,
-        urls = ["https://github.com/openzipkin/zipkin-api/archive/" + ZIPKINAPI_RELEASE + ".tar.gz"],
+        project_name = "Protobuf Rules for Bazel",
+        project_desc = "Protocol buffer rules for Bazel",
+        project_url = "https://github.com/bazelbuild/rules_proto",
+        version = "40298556293ae502c66579620a7ce867d5f57311",
+        sha256 = "aa1ee19226f707d44bee44c720915199c20c84a23318bb0597ed4e5c873ccbd5",
+        strip_prefix = "rules_proto-{version}",
+        urls = ["https://github.com/bazelbuild/rules_proto/archive/{version}.tar.gz"],
+        last_updated = "2020-08-17",
+        use_category = ["api"],
     ),
 )
+
+def _format_version(s, version):
+    return s.format(version = version, dash_version = version.replace(".", "-"), underscore_version = version.replace(".", "_"))
+
+# Interpolate {version} in the above dependency specs. This code should be capable of running in both Python
+# and Starlark.
+def _dependency_repositories():
```

Comment:
FYI I will refactor this with bazel/repository_locations.bzl in a followup where APIs are added to the dashboard.